owncloudApp/src/main/java/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -751,7 +751,7 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                 }
                 mPreferencesProvider.putBoolean(PassCodeActivity.PREFERENCE_SET_PASSCODE, true);
 
-                showSnackMessage(R.string.pass_code_stored);
+                //showSnackMessage(R.string.pass_code_stored);
```

Comment:
I'd recommend removing out commented lines.

Reply:
Same as before, unused class, but I'll try to make it cleaner from now on by removing lines instead of comment them, thanks!